src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="app">
+      <div class="app__wrapper">
```

Comment:
I believe we don't need these additional wrappers, so we increase the number of elements in the DOM tree, which is not good for performance.

Could we move (if needed) styles to the body?
Also, the class app is not the best choice , better use body or form etc